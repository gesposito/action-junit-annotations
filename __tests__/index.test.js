const { failingCasesFrom } = require("../index");

describe("Not found report", () => {
  let failingCases;
  test("Does not return failing cases", async () => {
    failingCases = await failingCasesFrom("./test-results/junit-not-found.xml");
    expect(failingCases).toMatchSnapshot();
  });
});
