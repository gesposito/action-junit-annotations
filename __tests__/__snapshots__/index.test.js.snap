// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not found report Does not return failing cases 1`] = `Array []`;
